plugins/sql-language/src/main/java/org/polypheny/db/sql/language/validate/SqlValidatorImpl.java

```diff
@@ -174,7 +173,7 @@
  */
 public class SqlValidatorImpl implements SqlValidatorWithHints {
 
-    NameMatcher nameMatcher = NameMatchers.withCaseSensitive( RuntimeConfig.RELATIONAL_CASE_SENSITIVE.getBoolean() );
+    NameMatcher nameMatcher = NameMatchers.withCaseSensitive( false );
```

Comment:
Why false? Shouldn't this use the case sensitivity setting of the namespace in which the entity is located? This gets a bit tricky for cross-model queries, but I think this can be delegated to the snapshot.